java/memory/memory-core/src/test/java/org/apache/arrow/memory/TestBaseAllocator.java

```diff
@@ -448,73 +443,73 @@ public ArrowBuf empty() {
   @Test
   public void testRootAllocator_listeners() throws Exception {
     CountingAllocationListener l1 = new CountingAllocationListener();
-    assertEquals(0, l1.getNumPreCalls());
-    assertEquals(0, l1.getNumCalls());
-    assertEquals(0, l1.getNumReleaseCalls());
-    assertEquals(0, l1.getNumChildren());
-    assertEquals(0, l1.getTotalMem());
+    Assert.assertEquals(0, l1.getNumPreCalls());
```

Comment:
nit: should we make `l1` -> `listner1`. I know it is not related to this PR `11` vs `l1`, just for the readability. And may be the following names too?

Reply:
Done.